torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6881,6 +6881,23 @@ def sample_inputs_allclose(op_info, device, dtype, requires_grad, **kwargs):
 
     return samples
 
+def sample_inputs_attn(op_info, device, dtype, requires_grad, **kwargs):
+    A = [10, 40]
```

Comment:
nit: these inputs are quite big. We usually try to use smaller things to keep CI fast.